### PR TITLE
Allow the bundle to use symfony 2.7 or 2.8 version

### PR DESCRIPTION
**Pull request by aliel**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,12 +17,12 @@
     "require": {
         "php": "^7.0",
         "graphaware/neo4j-php-client": "^4.6.4",
-        "symfony/framework-bundle": "^3.0",
+        "symfony/framework-bundle": "^2.7|^3.0",
```
Comment (Nyholm) [inline composer.json]: The official syntax is `"^2.7 || ^3.0"`

File: composer.json [hunk 2]
```diff
@@ -17,12 +17,12 @@
...
         "graphaware/neo4j-bolt": "^1.6"
     },
     "require-dev": {
         "phpunit/phpunit": "^5.4",
-        "symfony/symfony": "^3.0",
+        "symfony/symfony": "^2.7|^3.0",
```
Comment (Nyholm) [inline composer.json]: Same here. Use double pipe

**Comment by Nyholm** [comment]
Thank you

**Comment by aliel** [comment]
Hi, I reviewed the code and nothing to report.

**Comment by Nyholm** [comment]
Yes, So did I. Thanks

**Pull request merged**